processors/src/RoryClusterAnaProcessor.cxx

```diff
@@ -0,0 +1,677 @@
+    timesOnTrk_->GetXaxis()->SetTitle("Time (ns)");
+    timesOnTrk_->GetYaxis()->SetTitle("Hits"); 
+    timesOffTrk_->GetXaxis()->SetTitle("Time (ns)");
+    timesOffTrk_->GetYaxis()->SetTitle("Hits"); 
+
+    c1->DrawFrame(0.0,3000.0,150.0,7000.0);
+    c1->SetTitle("Times for All Clusters");
+    //std::cout<<"I AM IN THE FINAL STEP 2"<<std::endl; 
+    //std::cout<<layers_->GetEntries()<<std::endl;
+    times_->Draw();
+    c1->SaveAs("alltimes.png");     
+    c1->Clear();
+
+    c1->SetTitle("Times for On Track Clusters");
+    timesOnTrk_->Draw("e");
+    c1->SaveAs("ontimes.png");     
+    c1->Clear();
+    
+    c1->SetTitle("Times for Off Track Clusters");
+    timesOffTrk_->Draw("e");
+    c1->SaveAs("offtimes.png");     
+    c1->Clear();
+
+    times_->SetTitle("Cluster Times for all Cluster Cuts");
+    auto legend = new TLegend(0.3,0.8,.68,.9);
+    times_->SetLineColor(kCyan);
+    timesOnTrk_->SetLineColor(kGreen);
+    legend->AddEntry(times_,"Times");
+    legend->AddEntry(timesOnTrk_,"Times On Track");
+    legend->AddEntry(timesOffTrk_,"Times Off Track");
+    times_->Draw("e");
+    timesOnTrk_->Draw("same e");
+    timesOffTrk_->Draw("same e");
+    legend->Draw("same e");
+    c1->SaveAs("PutOnTogetherTimes.png");
+    c1->Clear();
+    return;
+}
+
+void RoryClusterAnaProcessor::Plot7(){
+    std::cout<<"I AM IN THE FINAL STEP"<<std::endl; 
+    TCanvas *c1 = new TCanvas("c");
+    gPad->SetLogy(true);
+    c1->cd();
+    timesNTD_->GetXaxis()->SetTitle("Times (ns)");
+    timesNTD_->GetYaxis()->SetTitle("Hits"); 
+    timesOnTrkNTD_->GetXaxis()->SetTitle("Time (ns)");
+    timesOnTrkNTD_->GetYaxis()->SetTitle("Hits"); 
+    timesOffTrkNTD_->GetXaxis()->SetTitle("Time (ns)");
+    timesOffTrkNTD_->GetYaxis()->SetTitle("Hits"); 
+
+    c1->DrawFrame(0.0,3000.0,150.0,7000.0);
+    c1->SetTitle("Times for All Clusters");
+    //std::cout<<"I AM IN THE FINAL STEP 2"<<std::endl; 
+    //std::cout<<layers_->GetEntries()<<std::endl;
+    timesNTD_->Draw("e");
+    c1->SaveAs("alltimesNTD.png");     
+    c1->Clear();
+
+    c1->SetTitle("Times for On Track Clusters");
+    timesOnTrkNTD_->Draw("e");
+    c1->SaveAs("ontimesNTD.png");     
+    c1->Clear();
+    
+    c1->SetTitle("Times for Off Track Clusters");
+    timesOffTrkNTD_->Draw("e");
+    c1->SaveAs("offtimesNTD.png");     
+    c1->Clear();
+
+    timesNTD_->SetTitle("Cluster Times for all Cluster Cuts");
+    auto legend = new TLegend(0.3,0.8,.68,.9);
+    timesNTD_->SetLineColor(kCyan);
+    timesOnTrkNTD_->SetLineColor(kGreen);
+    legend->AddEntry(timesNTD_,"Times");
+    legend->AddEntry(timesOnTrkNTD_,"Times On Track");
+    legend->AddEntry(timesOffTrkNTD_,"Times Off Track");
+    timesNTD_->Draw("e");
+    timesOnTrkNTD_->Draw("same e");
+    timesOffTrkNTD_->Draw("same e");
+    legend->Draw("same e");
+    c1->SaveAs("PutOnTogetherTimesNTD.png");
+    c1->Clear();
+    return;
+}
+
+void RoryClusterAnaProcessor::TrackPlot1(){
```

Comment:
this name is almost good. maybe remove the 1 because there is only one track plot as far as I can tell?

Reply:
done